document/CoreFormat.mdk

```diff
@@ -303,51 +316,116 @@ an analysis run. The value of `invocationInfo` would have to be combined with in
+                "hashes": [
+                    {
+                        "value": "b13ce2678a8807ba0765ab94a0ecd394f869bc81",
+                        "algorithm": "SHA-256"
+                    }
+                ],
+                "tags": [ "entryPoint" ]
             }
         ]
-    },
+    }
+}
+```
+~
 
-    {
-        "uri": "file:///C:/Code/b.cc",
-        "hashes": [
+If the scanned file is nested within another file (for example, within a compressed container),
+then the first `fileReference` object in the array shall refer to the outermost container,
+the next one shall refer to a file nested immediately within that container, and so on,
+with the last element of the array referring to the most deeply nested file.
+
+In this case, the property name shall be the absolute URI of the outermost containing
+file, together with a fragment that describes the nesting of the scanned file within
+its parent or parents.
+If the scanned file is nested more than one level deep in the outermost containing file,
+the fragments representing each level of nesting may be combined in any way desired,
+as long as no two of the resulting property names are equivalent as defined in &sect;[#uriProperties].
+
+~ IsoNumberedExample
+Consider a Flash object contained in a word processing document which is in turned contained
+in a Zip archive.
+Suppose the path to the word processing document within the Zip archive is `/docs/intro.doc`,
+and suppose the Flash object is located at offset `0x17f2` from the start of the word processing
+document.
+Then one possible value for the property name within the `fileInfo` object would be:
+
+```
+file:///C:/Code/presentation.zip#/docs/intro.doc-0x17f2
+```
+
+Note that the following would _not_ be valid, because a URI fragment cannot contain a `#` character,
+and the property names within a `fileInfo` object are required to be valid URIs:
+
+```
+file:///C:/Code/presentation.zip#/docs/intro.doc#0x17f2
+```
+~
+
+~ IsoNumberedNote
+It need not be possible to use this URI to navigate directly to the nested scanned file.
+The information necessary to do that is specified in the `uri` properties of the
+`fileReference` objects in the array (&sect;[#fileReference-uri]).
+~
+
+~ IsoNumberedExample
+
+```
+runInfo: {
+    fileInfo: {
+        "file:///C:/Code/app.zip#/src/main.cc": [
             {
-                "value": "ffc654abaa74903dae1d2ff532078ec0a9d5f674",
-                "algorithm": "SHA-256"
-            }
+                "uri": "file:///C:/Code/app.zip"
+                "mimeType": "application/zip",
+
+            },
+            {
+                "uri": "#/src/main.cc"
+                "mimeType": "text/x-c",
+                "hashes": [
+                    {
+                        "value": "b13ce2678a8807ba0765ab94a0ecd394f869bc81",
+                        "algorithm": "SHA-256"
+                    }
+                ],
+                "tags": [ "entryPoint" ]
```

Comment:
It was just there to show how much stuff you could point to with a simple reference. But I've removed it.